resources/scripts/ttx_diff.py

```diff
@@ -650,14 +683,8 @@ def main(argv):
     root = Path(".").resolve()
     if root.name != "fontc":
         sys.exit("Expected to be at the root of fontc")
-    fontc_manifest_path = root / "fontc" / "Cargo.toml"
-    fontc_bin_path = root / "target" / "release" / "fontc"
-    otl_norm_manifest_path = root / "otl-normalizer" / "Cargo.toml"
-    otl_bin_path = root / "target" / "release" / "otl-normalizer"
-    build_crate(otl_norm_manifest_path)
-    build_crate(fontc_manifest_path)
-    assert otl_bin_path.is_file(), "failed to build otl-normalizer?"
-    assert fontc_bin_path.is_file(), "failed to build fontc?"
+
+    (fontc_bin_path, otl_bin_path) = get_fontc_and_normalizer_binary_paths(root)
```

Comment:
Seems clearer to have a function that takes in the crate name and the root path and call it twice

```python
fontc_file = get_binary(root, FLAGS.fontc)
# etc
```